### DISCUSSION TITLE
Cannot access Netbox via browser

### DISCUSSION THREAD

**Discussion by Sh4mmylad** [netbox-community/netbox#16005] [Help Wanted!] [answered]
Good morning,

I am attempting to get Netbox running for my organization.  After following the instructions in the how-to, I have been unsuccessful.  I am sure it has something to do with the HTTP server, though I am unsure how to fix it.

For now, I am just trying to get it to work via IP address.  Here is my apache config:
![image](https://github.com/netbox-community/netbox/assets/168865029/cd4271b9-30f5-44db-8d17-9ab108437718)

When I run a curl to the IP specified from the server, this is what I get:
![image](https://github.com/netbox-community/netbox/assets/168865029/34ae8e82-7169-4958-a8c0-7f52990396f1)

I've set the IP in allowed hosts:
![image](https://github.com/netbox-community/netbox/assets/168865029/41e5b4e1-b1bd-4743-be12-af2bf4b8e24c)

I am able to connect to the development server and login, so I am pretty sure it has something to do with the HTTP server side of the setup.  Any help/suggestions would be greatly appreciated.

**Comment by markkuleinio** [comment]
I'm not an Apache user anymore but at least your redirection has an issue: remove the %{} marking around the IP address.

**Comment by Sh4mmylad** [comment]
Thank you for your input!  I was using nginx originally but read that nginx can have issues with private IPs, so I switched it to Apache.

I have removed the %{} from the redirection and restarted apache2, however I am getting the same results:
![image](https://github.com/netbox-community/netbox/assets/168865029/d5127f01-3680-4b2a-9820-19fdbace0529)

**Reply by Sh4mmylad** [reply]
Figured I would also supply statuses for netbox and apache2:

![image](https://github.com/netbox-community/netbox/assets/168865029/34722c85-0fba-46d5-a3a9-eabe34daba3c)
![image](https://github.com/netbox-community/netbox/assets/168865029/382f572c-bea6-4cb7-99d4-6639721d4581)

**Reply by Sh4mmylad** [reply]
Additionally, our organization has a Cisco firewall implemented; is it necessary to setup a rule to allow access to Netbox on the server I set it up on?

Is it required for the server that is running Netbox to be on the domain controller?

EDIT: I received an answer to the previous two questions; they were both false.

However, I am still unable to connect via inputting the server's IP in a browser's address bar.  Any help continued to be appreciated.

**Comment by Sh4mmylad** [comment, marked as answer]
I solved this issue by switching from Ubuntu 24.04 to Ubuntu 22.04.

It seems that Ubuntu 24 is not ready for NetBox without some intimate knowledge of what exactly is blocking the connection (it turns out that my PC was communicating with the server, but the server was not communicating back to the PC).
